lib/errors/DiscordRESTError.js

```diff
@@ -56,6 +56,12 @@ class DiscordRESTError extends Error {
             if(!Object.hasOwn(errors, fieldName) || fieldName === "message" || fieldName === "code") {
                 continue;
             }
+            if(fieldName === "_errors") {
+                messages = messages.concat(
+                    errors._errors.map((obj) => `${keyPrefix ? `${keyPrefix}: ` : ""}${obj.message}`)
+                );
+                continue;
+            }
             if(errors[fieldName]._errors) {
```

Comment:
small nit: the `continue;` statement might be redundant here
```suggestion
            } else if(errors[fieldName]._errors) {
```